Docs: Add example to extract a specific label value (#62711)
grafana/grafana: docs/sources/alerting/manage-notifications/template-notifications/using-go-templating-language.md

{{ end }}
```

> ## Use a specific label value

Reviewer: ```suggestion
## The index function
```

> You may only want to use some specific label values in your text, or use a longer description. 

Reviewer: ```suggestion
To print a specific annotation or label you can use the `index` function.
```

Reviewer 2: To print a specific annotation or label, use the 'index' function.

```

> This alert is in the folder {{ index .Labels "grafana_folder" }}

Reviewer: Would be great if we can put this in a range. A lot of users don't look at the context of code examples and instead just use them verbatim and then complain it doesn't work, so adding a `range` here should make it work if copied without reading the rest of the page.

```suggestion
{{ range .Alerts }}
This alert is in the folder {{ index .Labels "grafana_folder" }}
{{ end }}
```
